src/components/WalletModal/index.tsx

```diff
@@ -150,25 +150,25 @@ const ModalContent = (props: any) => {
                     {error instanceof UnsupportedChainId ? (
                         <>
                             <h5 className="text-center">{i18n._(t`Please connect to the appropriate network.`)}</h5>
-                            {/* Todo: Need to test how this will be handled with upcoming supported wallets*/} 
-                                <div className="flex flex-row justify-around pt-2 mt-5 align-center">
-                                    <ButtonAction
-                                        size="sm"
-                                        width="40%"
-                                        onClick={handleEthereumNetworkSwitch}
-                                        borderRadius="6px"
-                                    >
-                                        {i18n._(t`ETHEREUM`)}
-                                    </ButtonAction>
-                                    <ButtonAction
-                                        size="sm"
-                                        width="40%"
-                                        onClick={handleFuseNetworkSwitch}
-                                        borderRadius="6px"
-                                    >
-                                        {i18n._(t`FUSE`)}
-                                    </ButtonAction>
-                                </div>
+                            {/* Todo: Need to test how this will be handled with upcoming supported wallets*/}
+                            <div className="flex flex-row justify-around pt-2 mt-5 align-center">
+                                <ButtonAction
```

Comment:
couldn't we use action button from gooddesign ? just pass required chain id and noop as handler (as we want just switch without running any action)